### ISSUE TITLE
Cannot download ngc models in demo/BERT at release/7.2

### ISSUE THREAD

**Issue by david-macleod** [NVIDIA/TensorRT#869]
## Description

When following the instructions to reproduce the benchmarks in the demo/BERT readme I get stuck when trying to download models from NGC. For example:

```
bash scripts/download_model.sh large fp16 384 v2
Error: nvidia/bert_tf_v2_large_fp16_384:2 could not be found.
```

This error occurs for all configurations I have tried, including the defaults. I have also tried browsing NGC online catalogue to find the models manually but I cannot find the tags the `download_model.sh` script is searching for.

Thanks

## Environment

**TensorRT Version**: 7.2.1
**GPU Type**: T4
**Nvidia Driver Version**: 455.32.00
**CUDA Version**:  11.1
**CUDNN Version**: 8.06
**Operating System + Version**: Ubuntu 18.04
**Python Version (if applicable)**: 
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**: demo/BERT container

**Comment by mk-nvidia** [comment]
@david-macleod Yes it's a bug on our side, we're working to re-upload these checkpoints soon. Sorry for the inconvenience.

**Comment by david-macleod** [comment]
Thanks for the quick reply, is there any sort of estimated time frame for this?

**Comment by samringer** [comment]
I'm also seeing this exact error as well!

**Comment by willsq** [comment]
Me too - thanks for the reply, is there a blocker on this that might affect timescales?

**Comment by rajeevsrao** [comment]
Sorry for the inconvenience. Some of our NGC checkpoints were deleted - we will upload the new checkpoints by tomorrow (11/4) morning. I will post an update here as soon as that happens.

**Comment by rajeevsrao** [comment]
@david-macleod @samringer @willsq FYI fix posted to master and release/7.2 branches today:
https://github.com/NVIDIA/TensorRT/commit/f1d38b5456e15e2683a4513ea3cb14ed6d94cf53
